concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-32	32	Schnee	Schnee	snow	784	SNOW
+Steinthal-1875-1565-33	33	Hagel	Hagel	hail	609	HAIL
+Steinthal-1875-1565-34	34	Reif	Reif	hoar frost	2942	HOARFROST
+Steinthal-1875-1565-35	35	Eis schmilzt, thaut.	Eis schmilzt.	Ice melts.		
+Steinthal-1875-1565-36	36	Eis schmilzt, thaut.	Eis thaut.	Ice thaws.		
+Steinthal-1875-1565-37	37	Tag, bricht an, geht zu Ende	Tag	day	1225	DAY (NOT NIGHT)
+Steinthal-1875-1565-38	38	Tag, bricht an, geht zu Ende	Tag bricht an.	daybreak		
+Steinthal-1875-1565-39	39	Tag, bricht an, geht zu Ende	Tag geht zu Ende.	Day comes to an end.		
+Steinthal-1875-1565-40	40	Nacht, sie kommt herein	Nacht	night	1233	NIGHT
+Steinthal-1875-1565-41	41	Nacht, sie kommt herein	Nacht kommt herein.	Night falls.		
+Steinthal-1875-1565-42	42	Morgen	Morgen	morning	1339	MORNING
+Steinthal-1875-1565-43	43	Mittag	Mittag	noon	12	MIDDAY
+Steinthal-1875-1565-44	44	Abend	Abend	evening	1629	EVENING
+Steinthal-1875-1565-45	45	Morgenroth	Morgenroth	dawn	729	DAWN
+Steinthal-1875-1565-46	46	Abendröthe	Abendröthe	dusk	2297	DUSK
+Steinthal-1875-1565-47	47	Es ist finster.	Es ist finster.	It is gloomy.	1552	GLOOMY
+Steinthal-1875-1565-48	48	Finsterniss	Finsternis	gloom		
+Steinthal-1875-1565-49	49	dunkel, -heit	dunkel	dark	706	DARK
+Steinthal-1875-1565-50	50	dunkel, -heit	Dunkelheit	darkness	11	DARKNESS
+Steinthal-1875-1565-51	51	hell	hell	bright	679	BRIGHT
+Steinthal-1875-1565-52	52	Licht	Licht	light	630	LIGHT (RADIATION)
+Steinthal-1875-1565-53	53	Schatten	Schatten	shadow	1388	SHADE
+Steinthal-1875-1565-54	54	Jahr	Jahr	year	1226	YEAR
+Steinthal-1875-1565-55	55	Jahreszeit	Jahreszeit	season	691	SEASON
+Steinthal-1875-1565-56	56	Sommer (trockne Zeit)	Sommer	summer	1342	SUMMER
+Steinthal-1875-1565-57	57	Sommer (trockne Zeit)	trockne Zeit	dry season	452	DRY SEASON
+Steinthal-1875-1565-58	58	Winter (Regenzeit)	Winter	winter	1471	WINTER
+Steinthal-1875-1565-59	59	Winter (Regenzeit)	Regenzeit	wet season	453	RAINY SEASON
+Steinthal-1875-1565-60	60	- ist nun vorüber.	Sommer, Winter ist nun vorüber.	Summer, winter is over now.		
+Steinthal-1875-1565-61	61	Herbst (Erntezeit)	Herbst	autumn	1276	AUTUMN
+Steinthal-1875-1565-62	62	Herbst (Erntezeit)	Erntezeit	harvest season	3327	AUTUMN OR HARVEST SEASON
+Steinthal-1875-1565-63	63	Frühling (Saatzeit)	Frühling	spring	1537	SPRINGTIME
+Steinthal-1875-1565-64	64	Frühling (Saatzeit)	Saatzeit	sowing time		
+Steinthal-1875-1565-65	65	heute	heute	today	1283	TODAY
+Steinthal-1875-1565-66	66	gestern	gestern	yesterday	1174	YESTERDAY
+Steinthal-1875-1565-67	67	vorgestern	vorgestern	day before yesterday	1180	DAY BEFORE YESTERDAY
+Steinthal-1875-1565-68	68	morgen	morgen	tomorrow	1329	TOMORROW
+Steinthal-1875-1565-69	69	übermorgen	übermorgen	day after tomorrow	1179	DAY AFTER TOMORROW
+Steinthal-1875-1565-70	70	Woche	Woche	week	1469	WEEK
+Steinthal-1875-1565-71	71	Monat	Monat	month	1370	MONTH
+Steinthal-1875-1565-72	72	Namen der Wochentage und der Monate	Namen der Wochentage und der Monate	names of weekdays and months		
```

Comment:
True. And I was wondering about the months and days as well. Since this list was intended for worldwide use, can we assume that languages generally have 12 months and 7 days which correspond to ours at least time-wise? This thought kept me from splitting them up, but maybe I'm overthinking it.

Reply:
But then we would lose some important concepts that are clearly implied in the category name. (And, consequently, I would have to collapse all categories, which would do the list no justice.) Weltgegenden are just cardinal directions, and the concept should be the same across languages who make use of it, given the underlying universal basis. Weekdays and months, however, could vary in number.

I agree that consistency is important, here was my principle for splitting the categories up or leaving them as they are: if there is a manageable number and the split-up items are deemed to be somewhat universal - like fingers, numbers, cardinal directions - I split them up. If the categories were too specific - like hunting, fishing, warfare equipment, weekdays and months - I left them as they were. Colours are debatable, they are split for now, but that could be changed.